src/discord-cluster-manager/launchers/github.py

```diff
@@ -70,7 +87,13 @@ async def run_submission(
 
         await status.push("⏳ Waiting for workflow to start...")
         logger.info("Waiting for workflow to start...")
-        await run.wait_for_completion(lambda x: self.wait_callback(x, status))
+
+        timeout = get_timeout(config)
```

Comment:
we should add one or two minutes to account for github setting up the container etc.

Reply:
done